components/staking-hero.js

```diff
@@ -76,51 +77,12 @@ const StakingHero = () => {
       <section className="staking_hero" id="home">
         <div className="container">
           <div className="staking_hero__headline">
-            <h1>Earn WETH rewards for staking PLR token</h1>
+            <h1>Unstake your PLR tokens</h1>
           </div>
-          {/* Current Stats */}
-          <div className="staking_hero__stats">
-            <div className="staking_hero__stats__detail">
-              <p>Total ETH Staked</p>
-              <h4 className="gradient_border">{totalETHStacked}</h4>
-            </div>
-            <div className="staking_hero__stats__detail">
-              <p>Current APY</p>
-              <h4 className="gradient_border">{currentAPY}</h4>
-            </div>
-            <p className="staking_hero__stats__status">Live°</p>
+          <div className="staking_hero__dapp">
+            <PlrDaoStakingBuilder defaultTransactionBlock={'PLR_DAO_STAKE'} shouldDisplayForm={true} />

```

Comment:
Can you check if this works please? We've had to change this to `<PlrDaoStakingBuilderUnstake />` due to some issues which is where the merge conflict is coming from